clients/src/main/java/org/apache/kafka/common/utils/LogContext.java

```diff
@@ -50,10 +52,14 @@ public String logPrefix() {
     private static class KafkaLogger implements Logger {
         private final Logger logger;
         private final String logPrefix;
+        private final String fqcn;
+        private final boolean isLocationAware;
```

Comment:
Maybe a cleaner approach is to provide two separate `Logger` implementations.

Reply:
@hachikuji I agree with you. I have added two different Logger implementations now. Also I have moved LogContext inside a separate 'logger' which is sub package of utils. Can you please review the changes?

Reply:
Sorry, I just missed the changes from LogContext. I have modified LogContext accordingly now. @hachikuji can you please review it now.